scout/server/blueprints/alignviewers/controllers.py

```diff
@@ -81,16 +81,19 @@ def make_igv_tracks(
         stop = stop or variant_obj["end"]
         chrom = chrom or variant_obj.get("chromosome")
 
-    if all([start, stop, chrom]):
-        chromosome = chrom.replace("MT", "M")
-        display_obj["locus"] = "chr{0}:{1}-{2}".format(chromosome, start, stop)
-
     # Set genome build for displaying alignments:
-    if "38" in str(case_obj.get("genome_build", "37")) or chromosome == "M":
+    if "38" in str(case_obj.get("genome_build", "37")) or chromosome == "MT":
         build = "38"
     else:
         build = "37"
 
+    if all([start, stop, chrom]):
+        chromosome = chrom
+        if build == "37" or chromosome == "MT":
```

Comment:
I think we need to add an explanation to this code because if I am confused now imagine in one year time..

The problem is only X and Y chromosomes? What about chrom M from cases in build 38?

Reply:
Sure, the hg38-for-hg19-MT move is way in the past for me too. The alias file option looks like The Preferred Way To Do It though. This works so perhaps we better save it; I'll make a separate PR to test the alias thing and see if it is really bugged, or it just didn't find the file for the 1kg_v37.

Reply:
And no, the problem is with M/MT as well, but we already had the exception to use hg38 for M...